### DISCUSSION TITLE
Image Modal Zoom

### DISCUSSION THREAD

**Discussion by bobossler** [Lednerb/bilberry-hugo-theme#306] [Q&A] [answered]
I am trying out the Bilberry theme and really liking it. However, I cannot figure out how to make the Image Modal Zoom feature work. I have figured out Galleries and have automatic resizing turned on, but the large images do not open in a lightbox. The example/demo site doesn't seem to have large images, so I couldn't find it there. Does anybody else have it working? If so, any guidance would be appreciated.
I'm not on Discord; that's why I'm posting here. I'll give it a few days and if there are no responses, I will attempt to figure out Discord.

Thanks!

**Comment by igor-baiborodine** [comment, marked as answer]
@bobossler, I made a quick test with the Bilberry Sandbox. 

Here is the article that contains a large image:
https://www.bilberry-sandbox.kiroule.com/article/test-image-modal-zoom/

You can see the source code here: 
article: https://github.com/igor-baiborodine/bilberry-hugo-theme-sandbox/blob/master/content/article/test-image-modal-zoom.md
image: https://github.com/igor-baiborodine/bilberry-hugo-theme-sandbox/blob/master/static/img/content/article/test-image-modal-zoom/russian-winter.jpg

**Reply by bobossler** [reply]
Ah, ok, I see two differences in what I had tried. I was using a Post type of gallery, not article. Also, I was looking for it to work from within the slider. 

I'll have to rethink my organization and what I'm trying to do. I really like your theme and want to stick with it.

Thanks for the quick reply!!!

**Comment by mandarvaze** [comment]
@igor-baiborodine @bobossler Is it possible to make this work with Post type Gallery (rather than Article) ?

For Post type Gallery, the images are shown in the List view as well. But for Article type, image is visible only when user clicks on the post as views it as a single Post.

**Reply by igor-baiborodine** [reply]
@mandarvaze, I created discussion https://github.com/Lednerb/bilberry-hugo-theme/discussions/324 for this in the **Ideas** category.
